integrations/tensorflow/iree_tf_compiler/iree-tf-opt-main.cpp

```diff
@@ -41,8 +41,8 @@ int main(int argc, char **argv) {
 
   // Select IREE input passes.
   mlir::iree_compiler::registerCommonInputConversionPasses();
-  mlir::iree_compiler::registerMHLOConversionPasses();
   mlir::iree_compiler::registerTOSAConversionPasses();
+  mlir::iree_compiler::MHLO::registerMHLOConversionPasses();
```

Comment:
Does the order of these matter in some way?

Reply:
No. Just my brain sorting things lexographically.